src/test/resources/tema7catalinaz/suite.xml

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd" >
```

Comment:
Use CURAHealthcare from tests/Tema7 package and not the one from teorie/TestNg package. Thanks!